api-server/src/events/events.authenticated.controller.ts

```diff
@@ -21,6 +23,16 @@ export default class EventsAuthenticatedController {
 
     constructor(private readonly eventsService: EventsService){}
 
+    // this probably shouldn't be needed long term, we should just fetch this with the event
+    @Get('/admin-metadata')
+    @ApiOperation({ summary: 'Get all admin metadata for all events' })
+    @ApiImplicitParam({ name: 'id', type: String })
```

Comment:
Is this annotation supposed to be here? Doesn't look like `id` is used.

Reply:
nope, cpy/paste fail, good catch